src/transformStateWithClones.js

```diff
@@ -7,7 +7,37 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const result = [];
+  const stateVersion = { ...state };
+
```

Comment:
```const ADD = 'addProperties';```

Consider using constants instead of magic strings

Reply:
✅ Done